webpack.dev.js

```diff
@@ -0,0 +1,28 @@
+const merge = require('webpack-merge');
+const config = require('./webpack.config.js');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+module.exports = merge(config, {
+  mode: 'development',
+  devtool: 'source-map',
+  devServer: {
+    proxy: {
+      '/extensions': {
+        target: 'http://localhost:3001',
+        pathRewrite: { '^/extensions': '/public/extensions' }
+      }
+    },
+    port: 3001
+  },
+  plugins: [
+    ...config.plugins,
+    new WriteFilePlugin()
+  ],
+  // Watches for changes on snjs. Useful when developing snjs locally.
+  watchOptions: {
+    ignored: [
```

Comment:
Wait so is it watching or ignoring snjs? Is the objective to rebundle when snjs changes?

Reply:
> Wait so is it watching or ignoring snjs?

It's ignoring everything except `snjs`

> Is the objective to rebundle when snjs changes?

Yup.